### PR TITLE
Bug fix - Set equal height for resiliency and capacity card

### PR DESCRIPTION
**Pull request by cloudbehl**
Screenshot from 2019-07-25 02-54-43

Signed-off-by: Ankush Behl <cloudbehl@gmail.com>

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/capacity-card/capacity-card.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.ceph-capacity-card__dashboard-card {
+    height: 100%;
+}
```
Comment (rawagner) [inline frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/capacity-card/capacity-card.scss]: missing new line

File: frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/data-resiliency/data-resiliency.scss [hunk 2]
```diff
@@ -32,3 +32,7 @@
 .ceph-data-resiliency__utilization-bar {
   text-align: left;
 }
+
+.ceph-data-resiliency__dashboard-card {
+  height: 100%;
+}
```
Comment (rawagner) [inline frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/data-resiliency/data-resiliency.scss]: missing new line

File: frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/capacity-card/capacity-card.scss (hunk 1 above)
Comment (cloudbehl) [reply frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/capacity-card/capacity-card.scss]: Done

File: frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/data-resiliency/data-resiliency.scss (hunk 2 above)
Comment (cloudbehl) [reply frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/data-resiliency/data-resiliency.scss]: Done

**Comment by spadgett** [comment]
We should make the card headers the same height too if possible even if there's a dropdown

File: frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/data-resiliency/data-resiliency.scss [hunk 3]
```diff
@@ -32,3 +32,7 @@
 .ceph-data-resiliency__utilization-bar {
   text-align: left;
 }
+
+.ceph-data-resiliency__dashboard-card {
```
Comment (spadgett) [inline frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/data-resiliency/data-resiliency.scss]: alphabetize declarations

File: frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/capacity-card/capacity-card.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+.ceph-capacity-card__dashboard-card {
+    height: 100%;
```
Comment (spadgett) [inline frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/capacity-card/capacity-card.scss]: 2 space indentation

File: frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/capacity-card/capacity-card.scss [hunk 5]
```diff
@@ -0,0 +1,3 @@
+.ceph-capacity-card__dashboard-card {
```
Comment (spadgett) [inline frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/capacity-card/capacity-card.scss]: Should we consider a general solution that can be used in other contexts? I have a feeling we'll see this in more places as we build out these dashboards.

**Comment by spadgett** [comment]
@rhamilto might have better suggestions than me

**Comment by rawagner** [comment]
> We should make the card headers the same height too if possible even if there's a dropdown

this PR fixes header height https://github.com/openshift/console/pull/2149

**Pull request merged**